langchain/document_loaders/googledrive.py

```diff
@@ -30,11 +30,12 @@ class GoogleDriveLoader(BaseLoader, BaseModel):
     document_ids: Optional[List[str]] = None
     file_ids: Optional[List[str]] = None
     recursive: bool = False
+    file_types: Optional[str] = None
```

Comment:
Hi @UmerHA, for the type annotations, could you replace with an Optional[Sequence[str]] we want to capture the fact that several types are allowed.

```suggestion
    file_types: Optional[Sequence[str]] = None
```

Reply:
Done ✅